crates/api_common/src/utils.rs

```diff
@@ -43,7 +43,7 @@ use std::collections::HashSet;
 use tracing::warn;
 use url::{ParseError, Url};
 
-pub static AUTH_COOKIE_NAME: &str = "auth";
+pub static AUTH_COOKIE_NAME: &str = "jwt";
```

Comment:
I spose its fine. No need to log everyone out like we already did on lemmy.ml . I'll change this back in lemmy-ui